public/js/todoController.js

```diff
@@ -0,0 +1,17 @@
+var todoListApp = angular.module('todoList', []);
+
+todoListApp.controller('todoController', function($scope){
+
+  $scope.allTasks = [];
+
+  $scope.add = function(){
+  
+    $scope.allTasks.push({
+      text: $scope.newTask,
```

Comment:
The problem you're having with the text not being displayed may have something to do with the way your tasks are simply JavaScript objects, rather than prototypical objects? When you want to access the done status, you're referencing the object key whereas if you had this as a separate object (in its own file too) you could access the object's actual property.

Not sure what difference this really makes in practice, but it feels more extensible in the long run.